requirements.txt

```diff
@@ -20,3 +20,4 @@ six==1.11.0
 sklearn==0.0
 tabulate==0.8.2
 verlib==0.1
+torch
```

Comment:
Do we want to peg a version or something like a >= to this?

Reply:
[fdc5079](https://github.com/cmaclell/apprentice_learner_api/pull/35/commits/fdc5079a9f20a30f996533ac36b831d2397889b8)